src/args.rs

```diff
@@ -5,9 +5,10 @@ use solana_sdk::{pubkey::Pubkey, signature::Signer};
 use std::error::Error;
 
 pub struct DistributeTokensArgs<K> {
-    pub bids_csv: String,
+    pub input_csv: String,
+    pub allocations_csv: bool,
```

Comment:
How about making this if type `Option<String>` so that the only other change needed is to add `Option` to `bids_csv` and `dollars_per_sol`?